docs/index.md

```diff
@@ -12,6 +12,8 @@ This starter guide provides a broad overview of the platform to get you started
 
     To sign in to the Polly platform, you must have an account. Account credentials can be generated for both temporary, limited access as well as enterprise accounts. Reach out to our team at [polly@elucidata.io](mailto:polly@elucidata.io) for your login credentials.
 
+    We recommend using Chrome as a browser for accessing Polly. It is not fully compatible with other browsers.
```

Comment:
@Shreya-crypto Re-frame to something along the lines of experience has been optimized for Chrome and not other browsers